packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -58,6 +59,7 @@ class TerraService {
     global.expect.extend({ toBeAccessible });
 
     // Setup and expose global utility functions.
+    global.Terra.dispatchCustomEvent = dispatchCustomEvent;
```

Comment:
I'd personally probably rename this to `dispatchEvent`, but these are custom events so the name is descriptive as is without changes.

Reply:
yeah i think that's confusing. if someone wants to dispatch a regular event, they should just use the first class web api: `eventTarget.dispatchEvent`. the convenience of using this utility is firing a non pollyfilled custom event that's compatible with ie

Reply:
There are [CustomEvents](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent) in JavaScript so it makes both function name slightly confusing. I don't have any objections to keeping this as `dispatchCustomEvent`, it's bound to the `Terra` global so either function name is namespace safe